ament_cmake_python/cmake/ament_python_install_module.cmake

```diff
@@ -55,7 +57,16 @@ function(_ament_cmake_python_install_module module_file)
     FILES "${module_file}"
     DESTINATION "${destination}"
   )
-  # TODO optionally compile Python file
+  if(NOT ARG_SKIP_COMPILE)
+    # compile Python files
+    install(CODE
+      "execute_process(
+        COMMAND
+        \"${PYTHON_EXECUTABLE}\" \"-m\" \"compileall\"
```

Comment:
Question: should this accept a custom executable like [add_nose_test](https://github.com/ament/ament_cmake/pull/103/files#diff-29a991acf62a8fc4460ed46cdb27c46eR87) ? (same below)